src/ExternalModule.ts

```diff
@@ -110,11 +110,27 @@ export default class ExternalModule {
 						.map(name => `'${name}'`)
 						.join(', ')} and '${unused.slice(-1)}' are`;
 
+		const importersSet = new Set<string>()
+		for (const name of unused) {
+			const {importers, dynamicImporters} = this.declarations[name].module
+
+			if (Array.isArray(importers)) importers.forEach(v => importersSet.add(v))
+			if (Array.isArray(dynamicImporters)) dynamicImporters.forEach(v => importersSet.add(v))
+		}
+
+		const importersArray = Array.from(importersSet)
+
+		let importerList = ''
+		if (importersArray.length > 0) {
```

Comment:
It should not be possible that there is no importer, so I would assume we can also avoid this check.

Reply:
Resolved